# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/ipc/JSWindowActorParent.cpp
# Commit: 810b73719871
# Full Hash: 810b7371987139844429d0206f9da6a7701a1efc
# Author: Ryan Hunt <rhunt@eqrion.net>
# Date: 2019-05-23 04:41:59
# Regressor Bug: 1525720
# File Overlap Count: 1
# Description:
#   Bug 1525720, part 13 - Stop inheriting nsIRemoteTab interface in BrowserParent. r=nika
#   
#   This commit removes nsIRemoteTab as a parent class from BrowserParent,
#   so that BrowserHost is the only concrete implementation of nsIRemoteTab.
#   
# ==============================================================================

diff -r ee10a8254481 -r 810b73719871 dom/ipc/JSWindowActorParent.cpp
--- a/dom/ipc/JSWindowActorParent.cpp	Wed May 08 11:45:23 2019 -0500
+++ b/dom/ipc/JSWindowActorParent.cpp	Wed May 08 14:34:47 2019 -0500
@@ -72,7 +72,7 @@
 
   // Cross-process case - send data over WindowGlobalParent to other side.
   ClonedMessageData msgData;
-  RefPtr<BrowserParent> browserParent = mManager->GetRemoteTab();
+  RefPtr<BrowserParent> browserParent = mManager->GetBrowserParent();
   ContentParent* cp = browserParent->Manager();
   if (NS_WARN_IF(!aData.BuildClonedMessageDataForParent(cp, msgData))) {
     aRv.Throw(NS_ERROR_DOM_DATA_CLONE_ERR);